### DISCUSSION TITLE
How do I get a PR merged?

### DISCUSSION THREAD

**Discussion by kasperadk** [vercel/next.js#40264] [Help]
Hi. 

I opened this PR ([Add Enterspeed blog example](https://github.com/vercel/next.js/pull/35897)) back in April and got i approved in July.

What are the next steps in order to get this merged?

**Comment by icyJoseph** [comment]
I think it is still missing one more review IIRC, and if not, since it is an example, I guess it might be merged around the time a new version is released. 

I don't work with the Vercel team but I once fixed an example and it was merged, by someone else, after some time.
